integrations/docs/Integration-loading.md

```diff
@@ -0,0 +1,59 @@
+# Integration Loading
+
+**Loading Integrations into opensearch:**
+
+This phase describes the use case for a customer using Observability, it describes how such customer loads different Integrations so that it may be used to easily visualize and analyze existing data using pre-canned dashboards and visualizations.
+
+In the former part (Publishing An Integration) we defined the **Open Search Integration Verification Review Process.**
+
+The integrations passing this process can be available out of the box once the Observability plugin is loaded. This availability means that these Integrations can be packages together and assembled with the Observability solution.
+
+Once an Observability is distributed, it is pre-bundled with the verified Integrations. These integrations are packaged in a dedicated folder.
+
+**Integration Lifecycle**
+
+```
+                <-  Loading ->          
+                |                  |
+                |           <- Maintenance  
+       Ready2Ingest ->
+                                      <-  Ready2Read ->
+                                      |                 |
+                                Ingesting    ->   <-     Stale
+      
+```
```

Comment:
Can we just directly reference the `integration-loading-lifecycle.png` file here?

Reply:
yes